server/server.py

```diff
@@ -93,7 +93,7 @@ def robots_to_json(keys):
         robot = pickle.loads(redis_client.get(key))
         if robot is None:
             print("Database returned None for key {}".format(key))
-        time_stamp = json.dumps(robot.time_stamp, default=date_handler)
+        time_stamp = date_handler(robot.time_stamp)
```

Comment:
`json.dumps` added an extra pair of double quotes to the final JSON, which did nothing but adding complexity.